osc-domain/src/main/java/org/osc/core/broker/model/entities/virtualization/SecurityGroup.java

```diff
@@ -75,7 +75,7 @@
 
     @OneToMany(mappedBy = "securityGroup", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
     @OrderBy("type")
-    private Set<SecurityGroupMember> securityGroupMembers = new TreeSet<>();
+    private Set<SecurityGroupMember> securityGroupMembers = new HashSet<>();
```

Comment:
For entities coming from the db observe the OrderBy above. I believe the comparable wold still make sense if there is any piece of code that is initiating this field without db data, sorting it and expecting it would be by type. I am not sure if there is such case but if not I agree that the comparable can be removed.